### PR TITLE
Add Zeplin.app Latest

### PR DESCRIPTION
**Pull request by guerrero**

### CODE REVIEW

File: Casks/zeplin.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask :v1 => 'zeplin' do
+  version :latest
+  sha256 :no_check
+
+  # Download hostname differs from that of the homepage but there's a
+  # download link at https://zeplin.io/support.html#download
```
Comment (vitorgalvao) [inline Casks/zeplin.rb]: The comments needs to follwow the same pattern as the others.

File: Casks/zeplin.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  url 'https://zpl.io/download'
+  name 'Zeplin'
+  homepage 'https://zeplin.io'
+  license :freemium
```
Comment (vitorgalvao) [inline Casks/zeplin.rb]: Not `:freemium`, but `:gratis`. What matters is the app itself, not the business model of the company.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in d11b7af0443f110431c8b4f90d5f2e1e2f4c5af7. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by guerrero** [comment]
All the changes are clear, thanks for the detailed support, is much appreciated.

**Pull request closed without merging**